cdap-app-templates/cdap-etl/cdap-etl-archetypes/cdap-etl-transform-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -23,11 +23,17 @@
   <version>${version}</version>
   <packaging>jar</packaging>
 
-  <name>ETL Realtime Sink </name>
+  <name>CDAP Application</name>
```

Comment:
this is not a CDAP application right? i know its just the name used for this app, maybe custom-transform would be better?